pom.xml

```diff
@@ -5,7 +5,7 @@
 	<parent>
 		<groupId>net.imglib2</groupId>
 		<artifactId>pom-imglib2</artifactId>
-		<version>10.0.1</version>
+		<version>10.0.2</version>
```

Comment:
Why do we need the bump here? Is it for the `LoopBuilder`?

Reply:
Yes, it's because this makes `LoopBuilder` available.
I changed commit message, to include that information.